packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -450,6 +435,69 @@ export default class VTable {
         }
         return result;
     }
+
+    private setHTMLElementSizeInPx(element: HTMLElement, newWidth?: number, newHeight?: number) {
+        if (!!element) {
+            element.removeAttribute('width');
+            element.removeAttribute('height');
+            element.style.boxSizing = 'border-box';
+            element.style.width = `${
+                newWidth !== undefined ? newWidth : element.getBoundingClientRect().width
+            }px`;
+            element.style.height = `${
+                newHeight !== undefined ? newHeight : element.getBoundingClientRect().height
+            }px`;
+        }
+    }
+
+    public setEmptyTableCells() {
+        for (let i = 0, row; (row = this.table.rows[i]); i++) {
+            for (let j = 0, cell; (cell = row.cells[j]); j++) {
+                if (!cell.innerHTML) {
+                    cell.appendChild(document.createElement('br'));
+                }
+            }
+        }
+    }
+
+    public setTableCells() {
+        // measure and store the width/height into VCell
+        for (let i = 0, row; (row = this.table.rows[i]); i++) {
+            for (let j = 0, cell; (cell = row.cells[j]); j++) {
+                if (this.cells[i][j]) {
+                    const rect = cell.getBoundingClientRect();
+                    this.cells[i][j].width = rect.width;
+                    this.cells[i][j].height = rect.height;
+                }
+            }
+        }

```

Comment:
Done. I did this during resizing was to delay the shifting effect (mouse move), since now the shifting issue was resolved, I can just move this logic in ctor (mouse down).